runtime/doc/options.txt

```diff
@@ -3174,7 +3174,6 @@ A jump table for the options with a short description can be found at |Q_op|.
 <
 	For Mac OSX you can use something like this: >
 	    :set guifont=Monaco:h10
-<	Also see 'macatsui', it can help fix display problems.
```

Comment:
The `<` at the beginning of the line should be kept. If you're not already, make sure to edit the docs using `conceallevel=0`. I use this mapping, although there's probably many ways to do this:
```
nnoremap <silent> coc
      \ :set conceallevel=<C-r>=&conceallevel == 2 ? 0 : 2<CR><CR>
      \ :set conceallevel?<CR>

```